library/src/main/java/com/alamkanak/weekview/WeekViewEvent.java

```diff
@@ -106,4 +106,20 @@ public long getId() {
     public void setId(long id) {
         this.mId = id;
     }
+
+    @Override
+    public boolean equals(Object o) {
+        if (this == o) return true;
+        if (o == null || getClass() != o.getClass()) return false;
+
+        WeekViewEvent that = (WeekViewEvent) o;
+
+        return mId == that.mId;
+
+    }
+
+    @Override
+    public int hashCode() {
+        return (int) (mId ^ (mId >>> 32));
+    }
```

Comment:
He said he wants to check for the unicity of every appmt so I guess it is for that reason. Kind of the same idea as mine #109 (but he added the long appmt feature which is cool)